lib/Model.js

```diff
@@ -47,7 +47,8 @@ async function processCondition (req, options, model) {
         const val = options.conditionValues[k];
         const attr = model.$__.schema.attributes[k];
         if (attr) {
-          req.ExpressionAttributeValues[`:${k}`] = await attr.toDynamo(val, undefined, model, {'updateTimestamps': false});
+          const noSet = options.condition.startsWith('contains');
```

Comment:
Any opinion on this? Is this really the safest way to do this? Seems like it could create edge cases. I could be wrong tho.

Reply:
This _should_ be fine. I can't really imagine too many words that start with contains.

Reply:
There might be more edge cases (like, more cases in which the attribute should NOT be a set), but at least it catches this one. Also, from reading the documentation, I did not find another case. One could change it to check for 'contains(' instead of 'contains', but I found it sufficient like this. As @hweeks said, there are no other valid conditions that start with 'contains'.